blackbox/docs/sql/aggregation.txt

```diff
@@ -222,10 +222,10 @@ as a double value. Its single argument is the column name of a primitive numeric
     cr> select sum(name), kind from locations group by kind order by sum(name) desc;
     SQLActionException[UnsupportedFeatureException: unknown function: sum(string)]
 
-avg / mean
-==========
+``avg`` and ``mean``
+====================
 
-The ``avg`` or ``mean`` aggregation function returns the arithmetic mean, the *average*,
+The ``avg`` and ``mean`` aggregation function returns the arithmetic mean, the *average*,
```

Comment:
I debated this with myself! I think "and" is correct because they both do it, and that's what the sentence is talking about. the fact that you would use one or the other in an actual query, is a separate issue. I think

like "cheese and chocolate are both delicious"

perhaps it would make more sense then if I added a "both" to the sentence

thoughts?
versus "you can have cheese or chocolate"